shakenfist/constants.py

```diff
@@ -1,3 +1,8 @@
+# Note the most exciting constants ever
+KILOBYTE = 1024
+MEGABYTE = 1024 * 1024
+GIGABYTE = 1024 * 1024 * 1024
```

Comment:
I don't suppose I can convince you to throw style to the wind, and use `KiB`, `MiB` and `GiB` instead? `MEBIBYTE` looks stupid, but a "megabyte" could be one of 3 distinct values (a million bytes, 1000KiB or 1024KiB). This forces a new dev to check this file, making the named constant less useful than the magic number.

Reply:
Seems reasonable. Done.